### PR TITLE
add import order config file for Eclipse

### PR DESCRIPTION
**Pull request by outofthecave**

### CODE REVIEW

**Comment by juharris** [comment]
This is only for Eclipse right? Would you put Eclipse in the filename?

**Comment by outofthecave** [comment]
Changed the file name.

**Comment by juharris** [comment]
Hmm now it looks like this is only the import order when you use Eclipse...

**Comment by outofthecave** [comment]
So?  This file can only be imported into Eclipse, so this file only applies to Eclipse.  Idk if Idea has a way to export and import the import order.  Maybe it's part of the formatter config in Idea.

**Comment by juharris** [comment]
We have a separate file for IntelliJ

File: maluuba-eclipse.importorder [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Organize Import Order
+#Tue Dec 29 11:07:35 EST 2015
+0=\#java
+1=\#javax
+2=\#lombok
+3=\#org
+4=\#org.maluuba
+5=\#com
+6=\#com.maluuba
+7=java
```
Comment (juharris) [inline maluuba-eclipse.importorder]: Actually the Infra ordering puts a blank here for others, so when I export mine I get:
```
0=\#java
1=\#javax
2=\#lombok
3=\#org
4=\#org.maluuba
5=\#com
6=\#com.maluuba
7=\#
8=java
9=javax
10=lombok
11=org
12=org.maluuba
13=com
14=com.maluuba
15=
```

**Comment by outofthecave** [comment]
I don't understand the point you're trying to make, sorry.  What do you want the file name to be?

**Comment by juharris** [comment]
What about naming the file maluuba.eclipse.importorder?
maluuba for the group that uses the file
.eclipse because the file is for Eclipse
.importorder at the end because that's what Eclipse does.

File: maluuba-eclipse.importorder (hunk 1 above)
Comment (outofthecave) [inline maluuba-eclipse.importorder]: I made the file according to the [Wiki](http://wiki.maluuba.com/w/index.php?title=Getting_Started_with_NLP#Organize_Imports).  If we want something else, then we should change the Wiki too.

Comment (juharris) [inline maluuba-eclipse.importorder]: Good point, I updated the wiki.

**Pull request merged**